src/mds/journal.cc

```diff
@@ -2208,7 +2208,7 @@ void EUpdate::generate_test_instances(std::list<EUpdate*>& ls)
 
 void EUpdate::update_segment()
 {
-  auto&& segment = get_segment();
+  auto& segment = get_segment();
```

Comment:
Leave this. It's future-proof in case `get_segment` returns an rvalue.

Reply:
done